### PR TITLE
Fix file browser for non `en` language pages 

### PR DESCRIPTION
**Pull request by ghost**
Fixes: #23612

I propose that the text variable for the custom file browser follow the same logic as the color variables (see the beginning of the variables file).

And, in addition, establish a text by default (in english), so that the custom file browser works even if it uses an unconfigured language and even if the "lang" attribute is not present.

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -447,14 +447,18 @@ $custom-file-border-radius: $input-border-radius !default;
 $custom-file-box-shadow:    $input-box-shadow !default;
 $custom-file-button-color:  $custom-file-color !default;
 $custom-file-button-bg:     $input-group-addon-bg !default;
-$custom-file-text: (
+
+// stylelint-disable
```
Comment (XhmikosR) [inline scss/_variables.scss]: Why do you disable stylelint here exactly?

Comment (XhmikosR) [inline scss/_variables.scss]: Why do you disable stylelint here exactly?

**Review by XhmikosR — DISMISSED** [review]

File: scss/_variables.scss (hunk 1 above)
Comment (lucascono) [reply scss/_variables.scss]: Stylelint suggests that maps begin on separate lines. I could have put the parentheses in separate lines but as in the color variables it is disabled and so it was approved, I thought it was the right way. It can be re-enabled but to pass the test have to put all parentheses in separate lines. And it should also be done in color variables.

Comment (XhmikosR) [reply scss/_variables.scss]: Ok, thanks for the explanation. There must be some way to tweak the rule for this, but I didn't have the time to look at it.

Either way, can you just disable the specific stylelint error instead of all?

```
// stylelint-disable rule
...
//stylelint-enable rule
```

Comment (lucascono) [reply scss/_variables.scss]: Yes, of course, I do now.

File: scss/_variables.scss [hunk 2]
```diff
@@ -448,7 +448,8 @@ $custom-file-box-shadow:    $input-box-shadow !default;
 $custom-file-button-color:  $custom-file-color !default;
 $custom-file-button-bg:     $input-group-addon-bg !default;
 
-// stylelint-disable
+// stylelint-disable function-parentheses-newline-inside
```
Comment (XhmikosR) [inline scss/_variables.scss]: Can you just combine the rules with a comma between?

Comment (lucascono) [reply scss/_variables.scss]: I did not know it. Everyday learn something new. Thank you.

**Comment by lucascono** [comment]
@XhmikosR Can you cancel the last change? It was a mistake. It's for another PR.

File: docs/4.0/getting-started/download.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ toc: true
 
 Download ready-to-use compiled code for **Bootstrap v{{ site.current_version}}** to easily drop into your project, which includes:
 
-- Compiled and minified CSS bundles (default, grid-only, and Reboot-only)
+- Compiled and minified CSS bundles (see [comparative of css files]({{ site.baseurl }}/docs/{{ site.docs_version }}/getting-started/contents/#comparative-of-css-files))
```
Comment (XhmikosR) [inline docs/4.0/getting-started/download.md]: Wrong branch, please revert the unrelated changes and push them to a new branch which will only have those.

Comment (lucascono) [reply docs/4.0/getting-started/download.md]: Yes, I apologize. All the contributions I'm making to Bootstrap today is to learn how to use git. It will not happen again.

Comment (XhmikosR) [reply docs/4.0/getting-started/download.md]: No worries :)

The patches will be squashed into one when merge. /CC @mdo

**Comment by XhmikosR** [comment]
@mdo: we have #23632 for the same issue too. Choose which one to merge when you tackle this.

**Review by XhmikosR — CHANGES_REQUESTED** [review]
Wrong merge. Would be better if you didn't do any manual changes unless you are 100% sure.
